doc/manpages/faq.asciidoc

```diff
@@ -0,0 +1,92 @@
+Frequently Asked Questions
+--------------------------
+
+Is there going to be a Windows port of Kakoune ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As many features provided by UNIX systems would be missing, or if anything
+much less performant on a Windows system, the incentive to porting the
+project to this operating system is pretty low.
+
+Moreover, you can get pretty decent performance by using Kakoune on Cygwin
+(which is officially supported).
+
+Can you get rid of the `boost` dependency and just use std::regex ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `boost-regex` library provides use with several features that are
+heavily relied upon across several core scripts, and a few of them are
+not available in the standard `std::regex` implementations. Therefore,
+until the standard catches up with `boost` in terms of features,
+the latter will remain a hard -mandatory- dependency.
+
+Kakoune is very slow on big files, what can I do about it ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default build mode (set in the `Makefile` of the `src` directory
+of the project) is "debug", which makes it convenient to track issues
+but also affects performance. To disable the debug mode, recompile the
+project by setting the `debug` variable in `src/Makefile` to `no`.
+
+Can I use Kakoune as a pager ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.
+
+Are there any non-console based frontends available ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+No graphical frontend is currently officially maintained, you can however try experimental community-developed ones.
+
+How do I automatically indent code, as Vim does with `==` ?
```

Comment:
Actually like a lot of Vim command it's `=` but if you double the key it applies to the current line `==` ⇔ `V=`.